--- horde-3.2-alpha/config/registry.php.dist~	2007-08-27 12:39:20.810324487 +0300
+++ horde-3.2-alpha/config/registry.php.dist	2007-08-27 12:40:41.772154077 +0300
@@ -44,31 +44,31 @@
 // 'horde' stanza below.
 
 $this->applications['horde'] = array(
-    'fileroot' => dirname(__FILE__) . '/..',
+    'fileroot' => '/usr/share/horde',
     'webroot' => _detect_webroot(),
     'initial_page' => 'login.php',
     'name' => _("Horde"),
     'status' => 'active',
-    'templates' => dirname(__FILE__) . '/../templates',
+    'templates' => '/usr/share/horde/templates',
     'provides' => 'horde',
 );
 
 $this->applications['mimp'] = array(
-    'fileroot' => dirname(__FILE__) . '/../mimp',
+    'fileroot' => '/usr/share/horde/mimp',
     'webroot' => $this->applications['horde']['webroot'] . '/mimp',
     'name' => _("Mobile Mail"),
     'status' => 'notoolbar',
 );
 
 $this->applications['dimp'] = array(
-    'fileroot' => dirname(__FILE__) . '/../dimp',
+    'fileroot' => '/usr/share/horde/dimp',
     'webroot' => $this->applications['horde']['webroot'] . '/dimp',
     'name' => _("Dynamic Mail"),
     'status' => 'notoolbar',
 );
 
 $this->applications['imp'] = array(
-    'fileroot' => dirname(__FILE__) . '/../imp',
+    'fileroot' => '/usr/share/horde/imp',
     'webroot' => $this->applications['horde']['webroot'] . '/imp',
     'name' => _("Mail"),
     'status' => 'active',
@@ -165,7 +165,7 @@
 );
 
 $this->applications['nag'] = array(
-    'fileroot' => dirname(__FILE__) . '/../nag',
+    'fileroot' => '/usr/share/horde/nag',
     'webroot' => $this->applications['horde']['webroot'] . '/nag',
     'name' => _("Tasks"),
     'status' => 'active',
@@ -249,7 +249,7 @@
 );
 
 $this->applications['whups'] = array(
-    'fileroot' => dirname(__FILE__) . '/../whups',
+    'fileroot' => '/usr/share/horde/whups',
     'webroot' => $this->applications['horde']['webroot'] . '/whups',
     'name' => _("Tickets"),
     'status' => 'active',
@@ -258,7 +258,7 @@
 );
 
 $this->applications['luxor'] = array(
-    'fileroot' => dirname(__FILE__) . '/../luxor',
+    'fileroot' => '/usr/share/horde/luxor',
     'webroot' => $this->applications['horde']['webroot'] . '/luxor',
     'name' => _("X-Ref"),
     'status' => 'active',
@@ -271,7 +271,7 @@
 );
 
 $this->applications['klutz'] = array(
-    'fileroot' => dirname(__FILE__) . '/../klutz',
+    'fileroot' => '/usr/share/horde/klutz',
     'webroot' => $this->applications['horde']['webroot'] . '/klutz',
     'name' => _("Comics"),
     'status' => 'active',
@@ -280,7 +280,7 @@
 );
 
 $this->applications['occam'] = array(
-    'fileroot' => dirname(__FILE__) . '/../occam',
+    'fileroot' => '/usr/share/horde/occam',
     'webroot' => $this->applications['horde']['webroot'] . '/occam',
     'name' => _("Courses"),
     'status' => 'active',
@@ -288,7 +288,7 @@
 );
 
 $this->applications['mottle'] = array(
-    'fileroot' => dirname(__FILE__) . '/../mottle',
+    'fileroot' => '/usr/share/horde/mottle',
     'webroot' => $this->applications['horde']['webroot'] . '/mottle',
     'name' => _("MOTD"),
     'status' => 'active',
@@ -296,7 +296,7 @@
 );
 
 $this->applications['jonah'] = array(
-    'fileroot' => dirname(__FILE__) . '/../jonah',
+    'fileroot' => '/usr/share/horde/jonah',
     'webroot' => $this->applications['horde']['webroot'] . '/jonah',
     'name' => _("News"),
     'status' => 'active',
@@ -312,7 +312,7 @@
 );
 
 $this->applications['goops'] = array(
-    'fileroot' => dirname(__FILE__) . '/../goops',
+    'fileroot' => '/usr/share/horde/goops',
     'webroot' => $this->applications['horde']['webroot'] . '/goops',
     'name' => _("Search Engines"),
     'status' => 'active',
@@ -325,7 +325,7 @@
 );
 
 $this->applications['juno'] = array(
-    'fileroot' => dirname(__FILE__) . '/../juno',
+    'fileroot' => '/usr/share/horde/juno',
     'webroot' => $this->applications['horde']['webroot'] . '/juno',
     'name' => _("Accounting"),
     'status' => 'active',
@@ -333,7 +333,7 @@
 );
 
 $this->applications['midas'] = array(
-    'fileroot' => dirname(__FILE__) . '/../midas',
+    'fileroot' => '/usr/share/horde/midas',
     'webroot' => $this->applications['horde']['webroot'] . '/midas',
     'name' => _("Ads"),
     'status' => 'active',
@@ -341,7 +341,7 @@
 );
 
 $this->applications['hylax'] = array(
-    'fileroot' => dirname(__FILE__) . '/../hylax',
+    'fileroot' => '/usr/share/horde/hylax',
     'webroot' => $this->applications['horde']['webroot'] . '/hylax',
     'name' => _("Faxes"),
     'status' => 'active',
@@ -349,7 +349,7 @@
 );
 
 $this->applications['sesha'] = array(
-    'fileroot' => dirname(__FILE__) . '/../sesha',
+    'fileroot' => '/usr/share/horde/sesha',
     'webroot' => $this->applications['horde']['webroot'] . '/sesha',
     'name' => _("Inventory"),
     'status' => 'active',
@@ -361,7 +361,7 @@
 );
 
 $this->applications['thor'] = array(
-    'fileroot' => dirname(__FILE__) . '/../thor',
+    'fileroot' => '/usr/share/horde/thor',
     'webroot' => $this->applications['horde']['webroot'] . '/thor',
     'name' => _("Projects"),
     'status' => 'active',
@@ -370,7 +370,7 @@
 );
 
 $this->applications['rakim'] = array(
-    'fileroot' => dirname(__FILE__) . '/../rakim',
+    'fileroot' => '/usr/share/horde/rakim',
     'webroot' => $this->applications['horde']['webroot'] . '/rakim',
     'name' => _("Support"),
     'status' => 'active',
@@ -378,7 +378,7 @@
 );
 
 $this->applications['hermes'] = array(
-    'fileroot' => dirname(__FILE__) . '/../hermes',
+    'fileroot' => '/usr/share/horde/hermes',
     'webroot' => $this->applications['horde']['webroot'] . '/hermes',
     'name' => _("Time Tracking"),
     'status' => 'active',
@@ -406,7 +406,7 @@
 );
 
 $this->applications['gollem'] = array(
-    'fileroot' => dirname(__FILE__) . '/../gollem',
+    'fileroot' => '/usr/share/horde/gollem',
     'webroot' => $this->applications['horde']['webroot'] . '/gollem',
     'name' => _("File Manager"),
     'status' => 'active',
@@ -422,7 +422,7 @@
 );
 
 $this->applications['passwd'] = array(
-    'fileroot' => dirname(__FILE__) . '/../passwd',
+    'fileroot' => '/usr/share/horde/passwd',
     'webroot' => $this->applications['horde']['webroot'] . '/passwd',
     'name' => _("Password"),
     'status' => 'active',
@@ -430,7 +430,7 @@
 );
 
 $this->applications['jeta'] = array(
-    'fileroot' => dirname(__FILE__) . '/../jeta',
+    'fileroot' => '/usr/share/horde/jeta',
     'webroot' => $this->applications['horde']['webroot'] . '/jeta',
     'name' => _("SSH"),
     'status' => 'active',
@@ -452,7 +452,7 @@
 );
 
 $this->applications['agora'] = array(
-    'fileroot' => dirname(__FILE__) . '/../agora',
+    'fileroot' => '/usr/share/horde/agora',
     'webroot' => $this->applications['horde']['webroot'] . '/agora',
     'name' => _("Forums"),
     'status' => 'active',
@@ -461,7 +461,7 @@
 );
 
 $this->applications['ulaform'] = array(
-    'fileroot' => dirname(__FILE__) . '/../ulaform',
+    'fileroot' => '/usr/share/horde/ulaform',
     'webroot' => $this->applications['horde']['webroot'] . '/ulaform',
     'name' => _("Forms"),
     'status' => 'active',
@@ -469,7 +469,7 @@
 );
 
 $this->applications['volos'] = array(
-    'fileroot' => dirname(__FILE__) . '/../volos',
+    'fileroot' => '/usr/share/horde/volos',
     'webroot' => $this->applications['horde']['webroot'] . '/volos',
     'name' => _("Guestbook"),
     'status' => 'active',
@@ -477,7 +477,7 @@
 );
 
 $this->applications['ansel'] = array(
-    'fileroot' => dirname(__FILE__) . '/../ansel',
+    'fileroot' => '/usr/share/horde/ansel',
     'webroot' => $this->applications['horde']['webroot'] . '/ansel',
     'name' => _("Photos"),
     'status' => 'active',
@@ -486,7 +486,7 @@
 );
 
 $this->applications['scry'] = array(
-    'fileroot' => dirname(__FILE__) . '/../scry',
+    'fileroot' => '/usr/share/horde/scry',
     'webroot' => $this->applications['horde']['webroot'] . '/scry',
     'name' => _("Polls"),
     'status' => 'active',
@@ -495,7 +495,7 @@
 );
 
 $this->applications['merk'] = array(
-    'fileroot' => dirname(__FILE__) . '/../merk',
+    'fileroot' => '/usr/share/horde/merk',
     'webroot' => $this->applications['horde']['webroot'] . '/merk',
     'name' => _("Shopping"),
     'status' => 'active',
@@ -504,7 +504,7 @@
 );
 
 $this->applications['swoosh'] = array(
-    'fileroot' => dirname(__FILE__) . '/../swoosh',
+    'fileroot' => '/usr/share/horde/swoosh',
     'webroot' => $this->applications['horde']['webroot'] . '/swoosh',
     'name' => _("SMS Messaging"),
     'status' => 'active',
@@ -513,7 +513,7 @@
 );
 
 $this->applications['wicked'] = array(
-    'fileroot' => dirname(__FILE__) . '/../wicked',
+    'fileroot' => '/usr/share/horde/wicked',
     'webroot' => $this->applications['horde']['webroot'] . '/wicked',
     'name' => _("Wiki"),
     'status' => 'active',
@@ -522,7 +522,7 @@
 );
 
 $this->applications['vilma'] = array(
-    'fileroot' => dirname(__FILE__) . '/../vilma',
+    'fileroot' => '/usr/share/horde/vilma',
     'webroot' => $this->applications['horde']['webroot'] . '/vilma',
     'name' => _("Mail Admin"),
     'status' => 'active',
@@ -521,7 +521,7 @@
         $webroot = preg_replace(';/' . $hordedir . '.*;', '/' . $hordedir, $path);
     } elseif (isset($_SERVER['PHP_SELF'])) {
         $webroot = preg_split(';/;', $_SERVER['PHP_SELF'], 2, PREG_SPLIT_NO_EMPTY);
-        $webroot = strstr(dirname(__FILE__), DIRECTORY_SEPARATOR . array_shift($webroot));
+        $webroot = strstr('/usr/share/horde', DIRECTORY_SEPARATOR . array_shift($webroot));
         if ($webroot !== false) {
             $webroot = preg_replace(array('/\\\\/', ';/config$;'), array('/', ''), $webroot);
         } elseif ($webroot === false) {
