--- horde-3.0.3/lib/api.php	2004-12-11 13:11:53.000000000 +0100
+++ horde-3.0.3-new/lib/api.php	2005-02-23 16:20:35.131403128 +0100
@@ -121,18 +121,18 @@
                      'link' => '%application%/admin/datatree.php',
                      'name' => _("_DataTree"),
                      'icon' => 'perms.png'),
-                 'phpshell' => array(
-                     'link' => '%application%/admin/phpshell.php',
-                     'name' => _("P_HP Shell"),
-                     'icon' => 'shell.png'),
-                 'sqlshell' => array(
-                     'link' => '%application%/admin/sqlshell.php',
-                     'name' => _("S_QL Shell"),
-                     'icon' => 'sql.png'),
-                 'cmdshell' => array(
-                     'link' => '%application%/admin/cmdshell.php',
-                     'name' => _("_CLI"),
-                     'icon' => 'shell.png'),
+#                 'phpshell' => array(
+#                     'link' => '%application%/admin/phpshell.php',
+#                     'name' => _("P_HP Shell"),
+#                     'icon' => 'shell.png'),
+#                 'sqlshell' => array(
+#                     'link' => '%application%/admin/sqlshell.php',
+#                     'name' => _("S_QL Shell"),
+#                     'icon' => 'sql.png'),
+#                 'cmdshell' => array(
+#                     'link' => '%application%/admin/cmdshell.php',
+#                     'name' => _("_CLI"),
+#                     'icon' => 'shell.png'),
                  );
 }
 
