diff -ur horde-3.0.9/lib/Horde/NLS.php horde-3.0.9-language/lib/Horde/NLS.php
--- horde-3.0.9/lib/Horde/NLS.php	2005-12-14 21:27:19.000000000 +0100
+++ horde-3.0.9-language/lib/Horde/NLS.php	2006-02-07 12:48:34.000000000 +0100
@@ -94,12 +94,13 @@
             $lang = NLS::select();
         }
 
+	$_SESSION['horde_language'] = $lang;
+
         if (isset($GLOBALS['language']) && $GLOBALS['language'] == $lang) {
             return;
         }
 
         $GLOBALS['language'] = $lang;
-        $_SESSION['horde_language'] = $lang;
 
         /* First try language with the current charset. */
         $lang_charset = $lang . '.' . NLS::getCharset();
