--- horde-3.0-rc3/config/registry.php.dist	2004-11-01 11:39:39.000000000 +0000
+++ horde-3.0-rc3.new/config/registry.php.dist	2004-12-12 23:00:36.489047024 +0000
@@ -56,17 +56,17 @@
 }
 
 $this->applications['horde'] = array(
-    'fileroot' => dirname(__FILE__) . '/..',
+    'fileroot' => '/usr/share/horde',
     'webroot' => $webroot,
     'initial_page' => 'login.php',
     'name' => _("Horde"),
     'status' => 'active',
-    'templates' => dirname(__FILE__) . '/../templates',
+    'templates' => '/usr/share/horde/templates',
     'provides' => 'horde'
 );
 
 $this->applications['imp'] = array(
-    'fileroot' => dirname(__FILE__) . '/../imp',
+    'fileroot' => '/usr/share/horde/imp',
     'webroot' => $this->applications['horde']['webroot'] . '/imp',
     'name' => _("Mail"),
     'status' => 'active',
@@ -81,7 +81,7 @@
 );
 
 $this->applications['ingo'] = array(
-    'fileroot' => dirname(__FILE__) . '/../ingo',
+    'fileroot' => '/usr/share/horde/ingo',
     'webroot' => $this->applications['horde']['webroot'] . '/ingo',
     'name' => _("Filters"),
     'status' => 'active',
@@ -129,7 +129,7 @@
 );
 
 $this->applications['turba'] = array(
-    'fileroot' => dirname(__FILE__) . '/../turba',
+    'fileroot' => '/usr/share/horde/turba',
     'webroot' => $this->applications['horde']['webroot'] . '/turba',
     'name' => _("Address Book"),
     'status' => 'active',
@@ -138,7 +138,7 @@
 );
 
 $this->applications['kronolith'] = array(
-    'fileroot' => dirname(__FILE__) . '/../kronolith',
+    'fileroot' => '/usr/share/horde/kronolith',
     'webroot' => $this->applications['horde']['webroot'] . '/kronolith',
     'name' => _("Calendar"),
     'status' => 'active',
@@ -154,7 +154,7 @@
 );
 
 $this->applications['mnemo'] = array(
-    'fileroot' => dirname(__FILE__) . '/../mnemo',
+    'fileroot' => '/usr/share/horde/mnemo',
     'webroot' => $this->applications['horde']['webroot'] . '/mnemo',
     'name' => _("Notes"),
     'status' => 'active',
@@ -163,7 +163,7 @@
 );
 
 $this->applications['nag'] = array(
-    'fileroot' => dirname(__FILE__) . '/../nag',
+    'fileroot' => '/usr/share/horde/nag',
     'webroot' => $this->applications['horde']['webroot'] . '/nag',
     'name' => _("Tasks"),
     'status' => 'active',
@@ -195,7 +195,7 @@
 );
 
 $this->applications['chora'] = array(
-    'fileroot' => dirname(__FILE__) . '/../chora',
+    'fileroot' => '/usr/share/horde/chora',
     'webroot' => $this->applications['horde']['webroot'] . '/chora',
     'name' => _("Version Control"),
     'status' => 'active',
@@ -283,7 +283,7 @@
 );
 
 $this->applications['klutz'] = array(
-    'fileroot' => dirname(__FILE__) . '/../klutz',
+    'fileroot' => '/usr/share/horde/klutz',
     'webroot' => $this->applications['horde']['webroot'] . '/klutz',
     'name' => _("Comics"),
     'status' => 'active',
