diff -uNr horde-2.2.7/config/registry.php.dist horde-2.2.7.new/config/registry.php.dist
--- horde-2.2.7/config/registry.php.dist	2003-10-08 12:31:51.000000000 +0200
+++ horde-2.2.7.new/config/registry.php.dist	2004-11-03 20:06:53.737928248 +0100
@@ -84,14 +84,14 @@
  */
 
 $this->applications['horde'] = array(
-    'fileroot' => dirname(__FILE__) . '/..',
+    'fileroot' => '/usr/share/horde',
     'webroot' => '/horde',
     'initial_page' => 'login.php',
     'icon' => '/horde/graphics/home.gif',
     'name' => _("Horde"),
     'allow_guests' => true,
     'status' => 'active',
-    'templates' => dirname(__FILE__) . '/../templates',
+    'templates' => '/usr/share/horde/templates',
     'cookie_domain' => $_SERVER['SERVER_NAME'],
     // ** If IE will be used to access Horde modules, you should read
     //    this discussion about the cookie_path setting (discussing issues
@@ -103,7 +103,7 @@
 );
 
 $this->applications['logout'] = array(
-    'fileroot' => dirname(__FILE__) . '/..',
+    'fileroot' => '/usr/share/horde',
     'webroot' => $this->applications['horde']['webroot'],
     'initial_page' => 'login.php?reason=logout',
     'icon' => $this->applications['horde']['webroot'] . '/graphics/logout.gif',
@@ -113,7 +113,7 @@
 );
 
 $this->applications['imp'] = array(
-    'fileroot' => dirname(__FILE__) . '/../imp',
+    'fileroot' => '/usr/share/horde/imp',
     'webroot' => $this->applications['horde']['webroot'] . '/imp',
     'icon' => $this->applications['horde']['webroot'] . '/imp/graphics/imp.gif',
     'name' => _("Mail"),
@@ -122,7 +122,7 @@
 );
 
 $this->applications['ingo'] = array(
-    'fileroot' => dirname(__FILE__) . '/../ingo',
+    'fileroot' => '/usr/share/horde/ingo',
     'webroot' => $this->applications['horde']['webroot'] . '/ingo',
     'icon' => $this->applications['horde']['webroot'] . '/ingo/graphics/ingo.gif',
     'name' => _("Filters"),
@@ -131,7 +131,7 @@
 );
 
 $this->applications['turba'] = array(
-    'fileroot' => dirname(__FILE__) . '/../turba',
+    'fileroot' => '/usr/share/horde/turba',
     'webroot' => $this->applications['horde']['webroot'] . '/turba',
     'icon' => $this->applications['horde']['webroot'] . '/turba/graphics/turba.gif',
     'name' => _("Address Book"),
@@ -140,7 +140,7 @@
 );
 
 $this->applications['kronolith'] = array(
-    'fileroot' => dirname(__FILE__) . '/../kronolith',
+    'fileroot' => '/usr/share/horde/kronolith',
     'webroot' => $this->applications['horde']['webroot'] . '/kronolith',
     'icon' => $this->applications['horde']['webroot'] . '/kronolith/graphics/kronolith.gif',
     'name' => _("Calendar"),
@@ -149,7 +149,7 @@
 );
 
 $this->applications['mnemo'] = array(
-    'fileroot' => dirname(__FILE__) . '/../mnemo',
+    'fileroot' => '/usr/share/horde/mnemo',
     'webroot' => $this->applications['horde']['webroot'] . '/mnemo',
     'icon' => $this->applications['horde']['webroot'] . '/mnemo/graphics/mnemo.gif',
     'name' => _("Memos"),
@@ -158,7 +158,7 @@
 );
 
 $this->applications['nag'] = array(
-    'fileroot' => dirname(__FILE__) . '/../nag',
+    'fileroot' => '/usr/share/horde/nag',
     'webroot' => $this->applications['horde']['webroot'] . '/nag',
     'icon' => $this->applications['horde']['webroot'] . '/nag/graphics/nag.gif',
     'name' => _("Tasks"),
@@ -167,7 +167,7 @@
 );
 
 $this->applications['chora'] = array(
-    'fileroot' => dirname(__FILE__) . '/../chora',
+    'fileroot' => '/usr/share/horde/chora',
     'webroot' => $this->applications['horde']['webroot'] . '/chora',
     'icon' => $this->applications['horde']['webroot'] . '/chora/graphics/chora.gif',
     'name' => _("CVS"),
@@ -176,7 +176,7 @@
 );
 
 $this->applications['klutz'] = array(
-    'fileroot' => dirname(__FILE__) . '/../klutz',
+    'fileroot' => '/usr/share/horde/klutz',
     'webroot' => $this->applications['horde']['webroot'] . '/klutz',
     'icon' => $this->applications['horde']['webroot'] . '/klutz/graphics/klutz.gif',
     'name' => _("Comics"),
