diff -ur horde-3.0.7/util/regex_test.php horde-3.0.7.new/util/regex_test.php
--- horde-3.0.7/util/regex_test.php	2005-09-30 02:51:52.000000000 +0000
+++ horde-3.0.7.new/util/regex_test.php	2005-11-29 17:19:28.466225000 +0000
@@ -8,7 +8,8 @@
  * $Horde: horde/util/regex_test.php,v 1.3.10.3 2005/09/30 02:51:52 selsky Exp $
  */
 
-require 'Horde/Util.php';
+@define('HORDE_BASE', dirname(__FILE__) . '/..');
+require_once HORDE_BASE . '/lib/core.php';
 $regexp = Util::getFormData('regexp');
 $subject = Util::getFormData('subject');
 $submit = Util::getFormData('submit');
