--- config/conf.php.dist~	2004-08-30 23:23:22.000000000 +0300
+++ config/conf.php.dist	2005-02-27 21:18:42.000000000 +0200
@@ -61,7 +61,7 @@
 $conf['session']['name'] = 'Horde';
 $conf['session']['cache_limiter'] = 'nocache';
 $conf['session']['timeout'] = 0;
-$conf['auth']['admins'] = array('Administrator');
+$conf['auth']['admins'] = array('');
 $conf['auth']['driver'] = 'auto';
 $conf['auth']['params'] = array('username' => 'Administrator');
 $conf['prefs']['driver'] = 'none';
