--- horde-3.0.3/scripts/ldap/horde.schema	2004-10-07 07:37:23.000000000 +0300
+++ ./scripts/ldap/horde.schema	2005-02-27 23:57:00.000000000 +0200
@@ -256,7 +256,7 @@
     AUXILIARY
     MAY ( mail $ hordePrefs $ impPrefs $ turbaPrefs $
         gollemPrefs $ kronolithPrefs $ mnemoPrefs $
-        trollPrefs $ nagPrefs $ klutzPrefs $
+        trollPrefs $ nagPrefs $ klutzPrefs $ choraPrefs $
         jonahPrefs $ hermesPrefs $ junoPrefs $
         treanPrefs $ whupsPrefs $ ingoPrefs $
 				geniePrefs $ scryPrefs $ anselPrefs $
