--- horde/lib/Api.php~	2011-04-09 01:01:50.000000000 +0300
+++ horde/lib/Api.php	2011-04-09 01:02:19.292467869 +0300
@@ -57,21 +57,21 @@
                 'name' => _("Sessions"),
                 'icon' => Horde_Themes::img('user.png')
             ),
-            'phpshell' => array(
-                'link' => '%application%/admin/phpshell.php',
-                'name' => _("P_HP Shell"),
-                'icon' => Horde_Themes::img('mime/php.png')
-            ),
-            'sqlshell' => array(
-                'link' => '%application%/admin/sqlshell.php',
-                'name' => _("S_QL Shell"),
-                'icon' => Horde_Themes::img('sql.png')
-            ),
-            'cmdshell' => array(
-                'link' => '%application%/admin/cmdshell.php',
-                'name' => _("_CLI"),
-                'icon' => Horde_Themes::img('shell.png')
-            )
+#            'phpshell' => array(
+#                'link' => '%application%/admin/phpshell.php',
+#                'name' => _("P_HP Shell"),
+#                'icon' => Horde_Themes::img('mime/php.png')
+#            ),
+#            'sqlshell' => array(
+#                'link' => '%application%/admin/sqlshell.php',
+#                'name' => _("S_QL Shell"),
+#                'icon' => Horde_Themes::img('sql.png')
+#            ),
+#            'cmdshell' => array(
+#                'link' => '%application%/admin/cmdshell.php',
+#                'name' => _("_CLI"),
+#                'icon' => Horde_Themes::img('shell.png')
+#            )
         );
 
         if (!empty($GLOBALS['conf']['activesync']['enabled'])) {
