diff -ur horde-3.1-rc1/lib/api.php horde-3.1-rc1.new/lib/api.php
--- horde-3.1-rc1/lib/api.php	2005-10-18 11:33:39.000000000 +0000
+++ horde-3.1-rc1.new/lib/api.php	2006-01-06 15:56:02.321528000 +0000
@@ -144,18 +144,18 @@
                      'link' => '%application%/admin/sessions.php',
                      'name' => _("Sessions"),
                      'icon' => 'user.png'),
-                 'phpshell' => array(
-                     'link' => '%application%/admin/phpshell.php',
-                     'name' => _("P_HP Shell"),
-                     'icon' => 'mime/php.png'),
-                 'sqlshell' => array(
-                     'link' => '%application%/admin/sqlshell.php',
-                     'name' => _("S_QL Shell"),
-                     'icon' => 'sql.png'),
-                 'cmdshell' => array(
-                     'link' => '%application%/admin/cmdshell.php',
-                     'name' => _("_CLI"),
-                     'icon' => 'shell.png'),
+#                 'phpshell' => array(
+#                     'link' => '%application%/admin/phpshell.php',
+#                     'name' => _("P_HP Shell"),
+#                     'icon' => 'shell.png'),
+#                 'sqlshell' => array(
+#                     'link' => '%application%/admin/sqlshell.php',
+#                     'name' => _("S_QL Shell"),
+#                     'icon' => 'sql.png'),
+#                 'cmdshell' => array(
+#                     'link' => '%application%/admin/cmdshell.php',
+#                     'name' => _("_CLI"),
+#                     'icon' => 'shell.png'),
                  );
 }
 
