--- ./lib/Horde/Serialize.php~	2005-08-04 22:00:24.000000000 +0300
+++ ./lib/Horde/Serialize.php	2005-10-08 21:12:24.013122735 +0300
@@ -327,10 +327,12 @@
         case SERIALIZE_GZ_ENCOD:
             return Util::extensionExists('zlib');
 
+        case SERIALIZE_SQLXML:
+            return file_exists("/usr/share/pear/XML/sql2xml.php");
+
         case SERIALIZE_NONE:
         case SERIALIZE_BASIC:
         case SERIALIZE_BASE64:
-        case SERIALIZE_SQLXML:
         case SERIALIZE_RAW:
         case SERIALIZE_URL:
         case SERIALIZE_UTF7:
