--- horde/config/mime_drivers.php~	2011-04-09 01:03:15.000000000 +0300
+++ horde/config/mime_drivers.php	2011-04-09 01:04:13.509122695 +0300
@@ -326,8 +326,7 @@
 
     /* RPM archive display. */
     'rpm' => array(
-        // Disabled by default
-        'disable' => true,
+        'disable' => false,
         'handles' => array(
             'application/x-rpm'
         ),
@@ -336,7 +335,7 @@
         ),
 
         // REQUIRED: Location of the rpm binary
-        'location' => '/usr/bin/rpm'
+        'location' => '/bin/rpm'
     ),
 
     /* Debian archive display. */
