--- horde-3.0-rc3/config/registry.php.dist      2004-11-01 11:39:39.000000000 +0000               
+++ ./config/registry.php.dist	2005-08-23 23:15:12.000000000 +0300
@@ -46,7 +46,7 @@
 // this.
 if (isset($_SERVER['PHP_SELF'])) {
     $webroot = preg_split(';/;', $_SERVER['PHP_SELF'], 2, PREG_SPLIT_NO_EMPTY);
-    $webroot = strstr(dirname(__FILE__), '/' . array_shift($webroot));
+    $webroot = strstr("/usr/share/horde", '/' . array_shift($webroot));
     if ($webroot !== false) {
         $webroot = preg_replace(';/config$;', '', $webroot);
     } else {
--- ./config/registry.php.dist~	2005-03-04 21:02:58.000000000 +0200
+++ ./config/registry.php.dist	2005-03-04 21:12:11.517027935 +0200
@@ -223,7 +223,7 @@
     'fileroot' => '/usr/share/horde/whups',
     'webroot' => $this->applications['horde']['webroot'] . '/whups',
     'name' => _("Tickets"),
-    'status' => 'inactive',
+    'status' => 'active',
     'provides' => 'tickets',
     'menu_parent' => 'devel'
 );
--- ./config/registry.php.dist~	2005-05-09 20:20:55.000000000 +0300
+++ ./config/registry.php.dist	2005-05-09 20:21:09.000000000 +0300
@@ -120,7 +120,7 @@
     'fileroot' => '/usr/share/horde/mimp',
     'webroot' => $this->applications['horde']['webroot'] . '/mimp',
     'name' => _("Mobile Mail"),
-    'status' => 'inactive'
+    'status' => 'active'
 );
 
 $this->applications['organizing'] = array(
--- ./config/registry.php.dist~	2005-05-09 20:23:33.000000000 +0300
+++ ./config/registry.php.dist	2005-05-09 20:52:13.000000000 +0300
@@ -488,7 +488,7 @@
     'fileroot' => '/usr/share/horde/wicked',
     'webroot' => $this->applications['horde']['webroot'] . '/wicked',
     'name' => _("Wiki"),
-    'status' => 'inactive',
+    'status' => 'active',
     'provides' => 'wiki',
     'menu_parent' => 'website'
 );
