--- horde-3.0-rc3/config/registry.php.dist      2004-11-01 11:39:39.000000000 +0000               
+++ ./config/registry.php.dist	2005-08-23 23:15:12.000000000 +0300
@@ -46,7 +46,7 @@
 // this.
 if (isset($_SERVER['PHP_SELF'])) {
     $webroot = preg_split(';/;', $_SERVER['PHP_SELF'], 2, PREG_SPLIT_NO_EMPTY);
-    $webroot = strstr(dirname(__FILE__), '/' . array_shift($webroot));
+    $webroot = strstr("/usr/share/horde", '/' . array_shift($webroot));
     if ($webroot !== false) {
         $webroot = preg_replace(';/config$;', '', $webroot);
     } else {
--- ./config/registry.php.dist~	2005-03-04 21:02:58.000000000 +0200
+++ ./config/registry.php.dist	2005-03-04 21:12:11.517027935 +0200
@@ -223,7 +223,7 @@
     'fileroot' => '/usr/share/horde/whups',
     'webroot' => $this->applications['horde']['webroot'] . '/whups',
     'name' => _("Tickets"),
-    'status' => 'inactive',
+    'status' => 'active',
     'provides' => 'tickets',
     'menu_parent' => 'devel'
 );
--- ./config/registry.php.dist~	2005-05-09 20:20:55.000000000 +0300
+++ ./config/registry.php.dist	2005-05-09 20:21:09.000000000 +0300
@@ -120,7 +120,7 @@
     'fileroot' => '/usr/share/horde/mimp',
     'webroot' => $this->applications['horde']['webroot'] . '/mimp',
     'name' => _("Mobile Mail"),
-    'status' => 'inactive'
+    'status' => 'active'
 );
 
 $this->applications['organizing'] = array(
--- ./config/registry.php.dist~	2005-05-09 20:23:33.000000000 +0300
+++ ./config/registry.php.dist	2005-05-09 20:52:13.000000000 +0300
@@ -488,7 +488,7 @@
     'fileroot' => '/usr/share/horde/wicked',
     'webroot' => $this->applications['horde']['webroot'] . '/wicked',
     'name' => _("Wiki"),
-    'status' => 'inactive',
+    'status' => 'active',
     'provides' => 'wiki',
     'menu_parent' => 'website'
 );
diff -ur horde-3.0.6-rc1-pre/config/registry.php.dist horde-3.0.6-rc1/config/registry.php.dist
--- horde-3.0.6-rc1-pre/config/registry.php.dist	2005-10-25 00:37:07.000000000 +0300
+++ horde-3.0.6-rc1/config/registry.php.dist	2005-10-25 00:37:18.000000000 +0300
@@ -57,24 +57,24 @@
 }
 
 $this->applications['horde'] = array(
-    'fileroot' => dirname(__FILE__) . '/..',
+    'fileroot' => '/usr/share/horde',
     'webroot' => $webroot,
     'initial_page' => 'login.php',
     'name' => _("Horde"),
     'status' => 'active',
-    'templates' => dirname(__FILE__) . '/../templates',
+    'templates' => '/usr/share/horde/templates',
     'provides' => 'horde'
 );
 
 $this->applications['mimp'] = array(
-    'fileroot' => dirname(__FILE__) . '/../mimp',
+    'fileroot' => '/usr/share/horde/mimp',
     'webroot' => $this->applications['horde']['webroot'] . '/mimp',
     'name' => _("Mobile Mail"),
     'status' => 'active'
 );
 
 $this->applications['imp'] = array(
-    'fileroot' => dirname(__FILE__) . '/../imp',
+    'fileroot' => '/usr/share/horde/imp',
     'webroot' => $this->applications['horde']['webroot'] . '/imp',
     'name' => _("Mail"),
     'status' => 'active',
@@ -82,7 +82,7 @@
 );
 
 $this->applications['ingo'] = array(
-    'fileroot' => dirname(__FILE__) . '/../ingo',
+    'fileroot' => '/usr/share/horde/ingo',
     'webroot' => $this->applications['horde']['webroot'] . '/ingo',
     'name' => _("Filters"),
     'status' => 'active',
@@ -91,7 +91,7 @@
 );
 
 $this->applications['sam'] = array(
-    'fileroot' => dirname(__FILE__) . '/../sam',
+    'fileroot' => '/usr/share/horde/sam',
     'webroot' => $this->applications['horde']['webroot'] . '/sam',
     'name' => _("Spam"),
     'status' => 'inactive',
@@ -102,7 +102,7 @@
 );
 
 $this->applications['forwards'] = array(
-    'fileroot' => dirname(__FILE__) . '/../forwards',
+    'fileroot' => '/usr/share/horde/forwards',
     'webroot' => $this->applications['horde']['webroot'] . '/forwards',
     'name' => _("Forwards"),
     'status' => 'inactive',
@@ -111,7 +111,7 @@
 );
 
 $this->applications['vacation'] = array(
-    'fileroot' => dirname(__FILE__) . '/../vacation',
+    'fileroot' => '/usr/share/horde/vacation',
     'webroot' => $this->applications['horde']['webroot'] . '/vacation',
     'name' => _("Vacation"),
     'status' => 'inactive',
@@ -132,7 +132,7 @@
 );
 
 $this->applications['turba'] = array(
-    'fileroot' => dirname(__FILE__) . '/../turba',
+    'fileroot' => '/usr/share/horde/turba',
     'webroot' => $this->applications['horde']['webroot'] . '/turba',
     'name' => _("Address Book"),
     'status' => 'active',
@@ -141,7 +141,7 @@
 );
 
 $this->applications['kronolith'] = array(
-    'fileroot' => dirname(__FILE__) . '/../kronolith',
+    'fileroot' => '/usr/share/horde/kronolith',
     'webroot' => $this->applications['horde']['webroot'] . '/kronolith',
     'name' => _("Calendar"),
     'status' => 'active',
@@ -157,7 +157,7 @@
 );
 
 $this->applications['mnemo'] = array(
-    'fileroot' => dirname(__FILE__) . '/../mnemo',
+    'fileroot' => '/usr/share/horde/mnemo',
     'webroot' => $this->applications['horde']['webroot'] . '/mnemo',
     'name' => _("Notes"),
     'status' => 'active',
@@ -166,7 +166,7 @@
 );
 
 $this->applications['nag'] = array(
-    'fileroot' => dirname(__FILE__) . '/../nag',
+    'fileroot' => '/usr/share/horde/nag',
     'webroot' => $this->applications['horde']['webroot'] . '/nag',
     'name' => _("Tasks"),
     'status' => 'active',
@@ -175,7 +175,7 @@
 );
 
 $this->applications['genie'] = array(
-    'fileroot' => dirname(__FILE__) . '/../genie',
+    'fileroot' => '/usr/share/horde/genie',
     'webroot' => $this->applications['horde']['webroot'] . '/genie',
     'name' => _("Wishlists"),
     'status' => 'inactive',
@@ -184,7 +184,7 @@
 );
 
 $this->applications['trean'] = array(
-    'fileroot' => dirname(__FILE__) . '/../trean',
+    'fileroot' => '/usr/share/horde/trean',
     'webroot' => $this->applications['horde']['webroot'] . '/trean',
     'name' => _("Bookmarks"),
     'status' => 'inactive',
@@ -198,7 +198,7 @@
 );
 
 $this->applications['chora'] = array(
-    'fileroot' => dirname(__FILE__) . '/../chora',
+    'fileroot' => '/usr/share/horde/chora',
     'webroot' => $this->applications['horde']['webroot'] . '/chora',
     'name' => _("Version Control"),
     'status' => 'active',
@@ -206,7 +206,7 @@
 );
 
 $this->applications['thor'] = array(
-    'fileroot' => dirname(__FILE__) . '/../thor',
+    'fileroot' => '/usr/share/horde/thor',
     'webroot' => $this->applications['horde']['webroot'] . '/thor',
     'name' => _("Projects"),
     'status' => 'inactive',
@@ -215,7 +215,7 @@
 );
 
 $this->applications['whups'] = array(
-    'fileroot' => dirname(__FILE__) . '/../whups',
+    'fileroot' => '/usr/share/horde/whups',
     'webroot' => $this->applications['horde']['webroot'] . '/whups',
     'name' => _("Tickets"),
     'status' => 'active',
@@ -224,7 +224,7 @@
 );
 
 $this->applications['luxor'] = array(
-    'fileroot' => dirname(__FILE__) . '/../luxor',
+    'fileroot' => '/usr/share/horde/luxor',
     'webroot' => $this->applications['horde']['webroot'] . '/luxor',
     'name' => _("X-Ref"),
     'status' => 'inactive',
@@ -232,7 +232,7 @@
 );
 
 $this->applications['nic'] = array(
-    'fileroot' => dirname(__FILE__) . '/../nic',
+    'fileroot' => '/usr/share/horde/nic',
     'webroot' => $this->applications['horde']['webroot'] . '/nic',
     'name' => _("Network"),
     'status' => 'inactive',
@@ -246,7 +246,7 @@
 );
 
 $this->applications['mottle'] = array(
-    'fileroot' => dirname(__FILE__) . '/../mottle',
+    'fileroot' => '/usr/share/horde/mottle',
     'webroot' => $this->applications['horde']['webroot'] . '/mottle',
     'name' => _("MOTD"),
     'status' => 'inactive',
@@ -254,7 +254,7 @@
 );
 
 $this->applications['jonah'] = array(
-    'fileroot' => dirname(__FILE__) . '/../jonah',
+    'fileroot' => '/usr/share/horde/jonah',
     'webroot' => $this->applications['horde']['webroot'] . '/jonah',
     'name' => _("News"),
     'status' => 'inactive',
@@ -263,7 +263,7 @@
 );
 
 $this->applications['occam'] = array(
-    'fileroot' => dirname(__FILE__) . '/../occam',
+    'fileroot' => '/usr/share/horde/occam',
     'webroot' => $this->applications['horde']['webroot'] . '/occam',
     'name' => _("Courses"),
     'status' => 'inactive',
@@ -271,7 +271,7 @@
 );
 
 $this->applications['goops'] = array(
-    'fileroot' => dirname(__FILE__) . '/../goops',
+    'fileroot' => '/usr/share/horde/goops',
     'webroot' => $this->applications['horde']['webroot'] . '/goops',
     'name' => _("Search Engines"),
     'status' => 'inactive',
@@ -279,7 +279,7 @@
 );
 
 $this->applications['klutz'] = array(
-    'fileroot' => dirname(__FILE__) . '/../klutz',
+    'fileroot' => '/usr/share/horde/klutz',
     'webroot' => $this->applications['horde']['webroot'] . '/klutz',
     'name' => _("Comics"),
     'status' => 'inactive',
@@ -288,7 +288,7 @@
 );
 
 $this->applications['troll'] = array(
-    'fileroot' => dirname(__FILE__) . '/../troll',
+    'fileroot' => '/usr/share/horde/troll',
     'webroot' => $this->applications['horde']['webroot'] . '/troll',
     'name' => _("Newsgroups"),
     'status' => 'inactive',
@@ -301,7 +301,7 @@
 );
 
 $this->applications['hermes'] = array(
-    'fileroot' => dirname(__FILE__) . '/../hermes',
+    'fileroot' => '/usr/share/horde/hermes',
     'webroot' => $this->applications['horde']['webroot'] . '/hermes',
     'name' => _("Time Tracking"),
     'status' => 'inactive',
@@ -310,7 +310,7 @@
 );
 
 $this->applications['juno'] = array(
-    'fileroot' => dirname(__FILE__) . '/../juno',
+    'fileroot' => '/usr/share/horde/juno',
     'webroot' => $this->applications['horde']['webroot'] . '/juno',
     'name' => _("Accounting"),
     'status' => 'inactive',
@@ -318,7 +318,7 @@
 );
 
 $this->applications['sesha'] = array(
-    'fileroot' => dirname(__FILE__) . '/../sesha',
+    'fileroot' => '/usr/share/horde/sesha',
     'webroot' => $this->applications['horde']['webroot'] . '/sesha',
     'name' => _("Inventory"),
     'status' => 'inactive',
@@ -331,7 +331,7 @@
 );
 
 $this->applications['accounts'] = array(
-    'fileroot' => dirname(__FILE__) . '/../accounts',
+    'fileroot' => '/usr/share/horde/accounts',
     'webroot' => $this->applications['horde']['webroot'] . '/accounts',
     'name' => _("My Account"),
     'status' => 'inactive',
@@ -339,7 +339,7 @@
 );
 
 $this->applications['gollem'] = array(
-    'fileroot' => dirname(__FILE__) . '/../gollem',
+    'fileroot' => '/usr/share/horde/gollem',
     'webroot' => $this->applications['horde']['webroot'] . '/gollem',
     'name' => _("File Manager"),
     'status' => 'active',
@@ -355,7 +355,7 @@
 );
 
 $this->applications['passwd'] = array(
-    'fileroot' => dirname(__FILE__) . '/../passwd',
+    'fileroot' => '/usr/share/horde/passwd',
     'webroot' => $this->applications['horde']['webroot'] . '/passwd',
     'name' => _("Password"),
     'status' => 'active',
@@ -363,7 +363,7 @@
 );
 
 $this->applications['jeta'] = array(
-    'fileroot' => dirname(__FILE__) . '/../jeta',
+    'fileroot' => '/usr/share/horde/jeta',
     'webroot' => $this->applications['horde']['webroot'] . '/jeta',
     'name' => _("SSH"),
     'status' => 'inactive',
@@ -376,7 +376,7 @@
 );
 
 $this->applications['giapeto'] = array(
-    'fileroot' => dirname(__FILE__) . '/../giapeto',
+    'fileroot' => '/usr/share/horde/giapeto',
     'webroot' => $this->applications['horde']['webroot'] . '/giapeto',
     'name' => _("CMS"),
     'status' => 'inactive',
@@ -385,7 +385,7 @@
 );
 
 $this->applications['midas'] = array(
-    'fileroot' => dirname(__FILE__) . '/../midas',
+    'fileroot' => '/usr/share/horde/midas',
     'webroot' => $this->applications['horde']['webroot'] . '/midas',
     'name' => _("Ads"),
     'status' => 'inactive',
@@ -393,7 +393,7 @@
 );
 
 $this->applications['agora'] = array(
-    'fileroot' => dirname(__FILE__) . '/../agora',
+    'fileroot' => '/usr/share/horde/agora',
     'webroot' => $this->applications['horde']['webroot'] . '/agora',
     'name' => _("Forums"),
     'status' => 'inactive',
@@ -402,7 +402,7 @@
 );
 
 $this->applications['ulaform'] = array(
-    'fileroot' => dirname(__FILE__) . '/../ulaform',
+    'fileroot' => '/usr/share/horde/ulaform',
     'webroot' => $this->applications['horde']['webroot'] . '/ulaform',
     'name' => _("Forms"),
     'status' => 'inactive',
@@ -410,7 +410,7 @@
 );
 
 $this->applications['volos'] = array(
-    'fileroot' => dirname(__FILE__) . '/../volos',
+    'fileroot' => '/usr/share/horde/volos',
     'webroot' => $this->applications['horde']['webroot'] . '/volos',
     'name' => _("Guestbook"),
     'status' => 'inactive',
@@ -418,7 +418,7 @@
 );
 
 $this->applications['ansel'] = array(
-    'fileroot' => dirname(__FILE__) . '/../ansel',
+    'fileroot' => '/usr/share/horde/ansel',
     'webroot' => $this->applications['horde']['webroot'] . '/ansel',
     'name' => _("Photos"),
     'status' => 'inactive',
@@ -427,7 +427,7 @@
 );
 
 $this->applications['scry'] = array(
-    'fileroot' => dirname(__FILE__) . '/../scry',
+    'fileroot' => '/usr/share/horde/scry',
     'webroot' => $this->applications['horde']['webroot'] . '/scry',
     'name' => _("Polls"),
     'status' => 'inactive',
@@ -436,7 +436,7 @@
 );
 
 $this->applications['rakim'] = array(
-    'fileroot' => dirname(__FILE__) . '/../rakim',
+    'fileroot' => '/usr/share/horde/rakim',
     'webroot' => $this->applications['horde']['webroot'] . '/rakim',
     'name' => _("Support"),
     'status' => 'inactive',
@@ -444,7 +444,7 @@
 );
 
 $this->applications['merk'] = array(
-    'fileroot' => dirname(__FILE__) . '/../merk',
+    'fileroot' => '/usr/share/horde/merk',
     'webroot' => $this->applications['horde']['webroot'] . '/merk',
     'name' => _("Shopping"),
     'status' => 'inactive',
@@ -452,7 +452,7 @@
 );
 
 $this->applications['swoosh'] = array(
-    'fileroot' => dirname(__FILE__) . '/../swoosh',
+    'fileroot' => '/usr/share/horde/swoosh',
     'webroot' => $this->applications['horde']['webroot'] . '/swoosh',
     'name' => _("SMS Messaging"),
     'status' => 'inactive',
@@ -461,7 +461,7 @@
 );
 
 $this->applications['wicked'] = array(
-    'fileroot' => dirname(__FILE__) . '/../wicked',
+    'fileroot' => '/usr/share/horde/wicked',
     'webroot' => $this->applications['horde']['webroot'] . '/wicked',
     'name' => _("Wiki"),
     'status' => 'active',
@@ -470,7 +470,7 @@
 );
 
 $this->applications['vilma'] = array(
-    'fileroot' => dirname(__FILE__) . '/../vilma',
+    'fileroot' => '/usr/share/horde/vilma',
     'webroot' => $this->applications['horde']['webroot'] . '/vilma',
     'name' => _("Mail Admin"),
     'status' => 'inactive',
